src/app/content/practiceQuestions/components/PQButton.tsx

```diff
@@ -13,4 +12,26 @@ const PQButton = styled(PlainButton)`
   padding: 0 3rem;
 `;
 
+// tslint:disable-next-line: variable-name
+const PQButton = styled(PlainButton)`
+  ${sharedStyles}
+`;
+
+// tslint:disable-next-line: variable-name
+export const PQInput = styled.input`
```

Comment:
Do we not want to use `input` for the submit buttons or do you mean replace `PQButton` with `<Button ...>`?

Reply:
button can be rendered as an input with `<Button component={<input type="text" />}` you might have to reset some of the input styles but at least you'll be using the actual button styles instead of re-creating them